charts/seed-bootstrap/charts/vpa/charts/runtime/templates/vpa-webhook-config.yaml

```diff
@@ -0,0 +1,51 @@
+{{- if .Values.admissionController.enabled }}
+{{- if not .Values.admissionController.registerWebhook }}
+---
+apiVersion: admissionregistration.k8s.io/v1
```

Comment:
This API is available only in k8s >=1.16?
Please use https://github.com/gardener/gardener/blob/master/charts/utils-templates/templates/_versions.tpl

Reply:
+1, use `v1beta1` please